tmt/steps/common.py

```diff
@@ -0,0 +1,93 @@
+import json
```

Comment:
There are other classes (and mixins) in `tmt.steps` already, is there a reason why `RebootCommon` deserves its own file?

Reply:
Hmm, I considered exactly this just at beginning. I also wasn't sure if separate file is the best solution. I compared what was already present in `tmt.steps` with `RebootCommon` and still the reboot related functionality seemed to me a bit specific, so I kept it separately. But I can change it for sure, I have no problem with it :+1:

Reply:
No strong feelings here, only the naming should the class stay in its own file, `common` would be puzzling with other common classes being in `__init__.py`. There is an issue with introducing more granularity, having smaller files, and this class may land in its own file anyway, but I for one would leave such a split to different PR. Starting with it right away might also be acceptable - I'm no canonical source, I just walk around and point out things :) - I'd just pick a different name, `reboot.py`, `rebootable.py`, ...?